pkg/pipeline/ingest/ingest_collector.go

```diff
@@ -57,6 +59,16 @@ type TransportWrapper struct {
 	c chan map[string]interface{}
 }
 
+var queueLength = operationalMetrics.NewGauge(prometheus.GaugeOpts{
+	Name: "ingest_collector_queue_length",
+	Help: "Queue length",
+})
+
+var linesProcessed = operationalMetrics.NewCounter(prometheus.CounterOpts{
+	Name: "ingest_collector_log_lines_processed",
```

Comment:
sorry, again to pick on naming, would name such as "ingest_collector_flow_logs_processed" be more appropriate since its processing flow logs.

Reply:
fixed